python/src/uagents/config.py

```diff
@@ -43,6 +45,21 @@
 DEFAULT_SEARCH_LIMIT = 100
 
 
+AgentType = Literal["hosted", "local", "mailbox", "proxy", "custom"]
```

Comment:
"local" means not connection with the agentverse whatsoever and "custom" only that it's bound to an agentverse account?

Reply:
Yes, exactly. There is probably some room for improvement in these names.